modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+| `neo4j_aura_cpu_usage`
+| _Gauge_
+| MAX
+
+| Memory Used
+| Memory in use by Neo4j.
+| `neo4j_aura_memory_usage`
+| _Gauge_
+| MAX
+
+| Memory Total
+| The total memory assigned to the instance.
+| `neo4j_aura_memory_limit`
+| _Gauge_
+| MAX
+
+| Storage Total
+| The total disk storage assigned to the instance.
+| `neo4j_aura_storage_limit`
+| _Gauge_
+| MAX
+
+| Heap Used
+| The percentage of configured heap memory in use. The heap space is used for query execution, transaction state, management of the graph etc. +
+The size needed for the heap is very dependent on the nature of the usage of Neo4j. For example, long-running queries, or very complicated queries, are likely to require a larger heap than simpler queries. Generally speaking, in order to aid performance, the heap should be large enough to sustain concurrent operations. +
+This value should not exceed 80% for long periods of time, short spikes can be normal. In case of performance issues you may have to tune your queries, and monitor their memory usage, in order to determine whether the heap needs to be increased. If the workload of Neo4j and performance of queries indicates that more heap space is required, consider increasing the size of your instance. This will help avoid unwanted pauses for garbage collection.
+| `neo4j_dbms_vm_heap_used_ratio`
+| _Gauge_
+| MAX
+
+| Page Cache Hit Ratio
+| The percentage of times data required during query execution was found in memory vs needing to be read from disk. Ideally the whole graph should fit into memory, and this should consistently be between 98% and 100%. If this value is consistently or significantly under 100%, check the page cache usage ratio to see if the graph is too large to fit into memory. A high amount of insert or update activity on a graph can also cause this value to change.
+| `neo4j_dbms_page_cache_hit_ratio_per_minute`
+| _Gauge_
+| AVG
+
+| Page Cache Usage Ratio
+| The percentage of the allocated page cache in use. If this is close to or at 100%, then it is likely that the hit ratio will start dropping, and you should consider increasing the size of your instance so that more memory is available for the page cache.
+| `neo4j_dbms_page_cache_usage_ratio`
+| _Gauge_
+| MIN
+
+| Bolt Connections Running
+| The total number of Bolt connections that are currently executing Cypher transactions and returning results. This is a set of snapshots over time and may appear to spike if workloads are all completed quickly.
+| `neo4j_dbms_bolt_connections_running`
+| _Gauge_
+| MAX
+
+| Bolt Connections Idle
+| The total number of Bolt connections that are connected to the Aura database but not currently executing Cypher or returning results.
+| `neo4j_dbms_bolt_connections_idle`
+| _Gauge_
+| MAX
+
+| Bolt Connections Closed
+| The total number of Bolt connections closed since startup. This includes both properly and abnormally ended connections. This value may drop if background maintenance is performed by Aura.
```

Comment:
```suggestion
| The total number of Bolt connections closed since startup. 
This includes both properly and abnormally ended connections. 
This value may drop if background maintenance is performed by Aura.
```